packages/expo-sharing/src/ExpoSharing.web.ts

```diff
@@ -13,6 +13,10 @@ export default {
     return 'ExpoSharing';
   },
   isAvailableAsync(): Promise<boolean> {
```

Comment:
```suggestion
  async isAvailableAsync(): Promise<boolean> {
```
then remove the `Promise.resolve(false)` in favor of `false`